From f34ca4bfca853f291fc9cb412111ad227b31498f Mon Sep 17 00:00:00 2001
From: Jayachandran C <jchandra@broadcom.com>
Date: Mon, 16 Jun 2014 11:39:18 +0530
Subject: nlm_adma: remove __devinit and __devexit attributes

This is no longer supported after 3.8, and removing these should not affect
3.7 kernel.
[Based on SDK 3.2]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
Signed-off-by: Nam Ninh <nam.ninh@windriver.com>

diff --git a/drivers/misc/netlogic/dtre/nlm_adma.c b/drivers/misc/netlogic/dtre/nlm_adma.c
index 79a0c5d..d59e855 100644
--- a/drivers/misc/netlogic/dtre/nlm_adma.c
+++ b/drivers/misc/netlogic/dtre/nlm_adma.c
@@ -1461,7 +1461,7 @@ static int alloc_initial_tx_desc_pool (struct nlm_adma_chan *nlm_chan, int count
 
 }
 
-static int __devinit nlm_adma_probe(struct platform_device *pdev)
+static int nlm_adma_probe(struct platform_device *pdev)
 {
 	struct nlm_adma_device *adev;
 	struct dma_device *dma_dev;
@@ -1688,7 +1688,7 @@ static int __devinit nlm_adma_probe(struct platform_device *pdev)
 }
 
 
-static int __devexit nlm_adma_remove(struct platform_device *dev)
+static int nlm_adma_remove(struct platform_device *dev)
 {
 	return 0;
 }
-- 
1.7.1

